build.gradle.kts

```diff
@@ -69,7 +67,7 @@ dependencies {
 tasks {
   test {
     useJUnitPlatform()
-    exclude("**/integration/*", "**/migrations/*")
```

Comment:
This is actually just removing a reference to a now redundant test package which no longer exists.